README.md

```diff
@@ -103,25 +104,26 @@ It should connect to Discord and display the currently logged-in application nam
 
 The expected flow for running a bust is:
 
-* Users submit songs into a channel.
-* All users join a voice channel or stage.
-* An admin runs `!list` to list all submitted songs and the order they will be played in.
-* An admin runs `!bust` to start the show. The bot will join the channel and begin playing songs in the order they were submitted.
-* Users comment on songs while they play.
-* An admin can run `!skip` at any time to skip the current song, or `!stop` to manually stop the show.
-* Once the last song has played, the bot will post a concluding message and leave the call.
+- Users submit songs into a channel.
+- All users join a voice channel or stage.
+- An admin runs `!list` to list all submitted songs and the order they will be played in.
+- An admin runs `!bust` to start the show. The bot will join the channel and begin playing songs in the order they were submitted.
+- Users comment on songs while they play.
+- An admin can run `!skip` at any time to skip the current song, or `!stop` to manually stop the show.
+- Once the last song has played, the bot will post a concluding message and leave the call.
```

Comment:
VSCode keeps converting these on save

Reply:
heh, you can turn this off by disabling the "Format on save" option in VSCode's preferences btw.